### ISSUE TITLE
Unable to install on python 3.7

### ISSUE THREAD

**Issue by sdabhi23** [zerodha/pykiteconnect#55]
I am getting the following error:

```bash
λ pip install kiteconnect
Collecting kiteconnect
  Using cached https://files.pythonhosted.org/packages/5a/dc/482cb13486946889b6a26abeb4036f5b8af19978251cc61f0ee45de4d833/kiteconnect-3.7.6.tar.gz
    Complete output from command python setup.py egg_info:
    Download error on https://pypi.org/simple/pytest-runner/: [WinError 10054] An existing connection was forcibly closed by the remote host -- Some packages may not be found!
    Couldn't find index page for 'pytest-runner' (maybe misspelled?)
    No local packages or working download links found for pytest-runner
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\Shrey\AppData\Local\Temp\pip-install-2mx7jwys\kiteconnect\setup.py", line 126, in <module>
        cmdclass={"install": install, "bdist_wheel": FakeBdist}
      File "d:\current_work\macd-auto\venv\lib\site-packages\setuptools\__init__.py", line 142, in setup
        _install_setup_requires(attrs)
      File "d:\current_work\macd-auto\venv\lib\site-packages\setuptools\__init__.py", line 137, in _install_setup_requires
        dist.fetch_build_eggs(dist.setup_requires)
      File "d:\current_work\macd-auto\venv\lib\site-packages\setuptools\dist.py", line 586, in fetch_build_eggs
        replace_conflicting=True,
      File "d:\current_work\macd-auto\venv\lib\site-packages\pkg_resources\__init__.py", line 780, in resolve
        replace_conflicting=replace_conflicting
      File "d:\current_work\macd-auto\venv\lib\site-packages\pkg_resources\__init__.py", line 1063, in best_match
        return self.obtain(req, installer)
      File "d:\current_work\macd-auto\venv\lib\site-packages\pkg_resources\__init__.py", line 1075, in obtain
        return installer(requirement)
      File "d:\current_work\macd-auto\venv\lib\site-packages\setuptools\dist.py", line 653, in fetch_build_egg
        return cmd.easy_install(req)
      File "d:\current_work\macd-auto\venv\lib\site-packages\setuptools\command\easy_install.py", line 673, in easy_install
        raise DistutilsError(msg)
    distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('pytest-runner')

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\Shrey\AppData\Local\Temp\pip-install-2mx7jwys\kiteconnect\
```

This error looks similar to [pytest-dev/pytest-runner #41](https://github.com/pytest-dev/pytest-runner/issues/41).

When I separately install pytest-runner first and then install kiteconnect I am facing the following issue:
```bash
Collecting zope.interface>=3.6.0; extra == "twisted" (from autobahn[twisted]>=17.10.1->kiteconnect->-r requirements.txt (line 4))                                                       
  Downloading https://files.pythonhosted.org/packages/da/08/726e3b0e3bd9912fb530f9864bf9a3af9f9f6a1dfd4cc7854ca14fdab441/zope.interface-4.6.0-cp36-cp36m-win_amd64.whl (133kB)          
    100% |████████████████████████████████| 143kB 212kB/s                                                                                                                               
Collecting Twisted>=12.1.0; extra == "twisted" (from autobahn[twisted]>=17.10.1->kiteconnect->-r requirements.txt (line 4))                                                             
  Downloading https://files.pythonhosted.org/packages/5d/0e/a72d85a55761c2c3ff1cb968143a2fd5f360220779ed90e0fadf4106d4f2/Twisted-18.9.0.tar.bz2 (3.1MB)                                 
    100% |████████████████████████████████| 3.1MB 180kB/s                                                                                                                               
    Complete output from command python setup.py egg_info:                                                                                                                              
    Download error on https://pypi.org/simple/incremental/: [WinError 10054] An existing connection was forcibly closed by the remote host -- Some packages may not be found!           
    Couldn't find index page for 'incremental' (maybe misspelled?)                                                                                                                      
    Download error on https://pypi.org/simple/: [WinError 10054] An existing connection was forcibly closed by the remote host -- Some packages may not be found!                       
    No local packages or working download links found for incremental>=16.10.1                                                                                                          
    Traceback (most recent call last):                                                                                                                                                  
      File "<string>", line 1, in <module>                                                                                                                                              
      File "C:\Users\Shrey\AppData\Local\Temp\pip-install-xtwuif77\Twisted\setup.py", line 20, in <module>                                                                              
        setuptools.setup(**_setup["getSetupArgs"]())                                                                                                                                    
      File "d:\current_work\macd-auto\macd-auto\lib\site-packages\setuptools\__init__.py", line 144, in setup                                                                           
        _install_setup_requires(attrs)                                                                                                                                                  
      File "d:\current_work\macd-auto\macd-auto\lib\site-packages\setuptools\__init__.py", line 139, in _install_setup_requires                                                         
        dist.fetch_build_eggs(dist.setup_requires)                                                                                                                                      
      File "d:\current_work\macd-auto\macd-auto\lib\site-packages\setuptools\dist.py", line 724, in fetch_build_eggs                                                                    
        replace_conflicting=True,                                                                                                                                                       
      File "d:\current_work\macd-auto\macd-auto\lib\site-packages\pkg_resources\__init__.py", line 782, in resolve                                                                      
        replace_conflicting=replace_conflicting                                                                                                                                         
      File "d:\current_work\macd-auto\macd-auto\lib\site-packages\pkg_resources\__init__.py", line 1065, in best_match                                                                  
        return self.obtain(req, installer)                                                                                                                                              
      File "d:\current_work\macd-auto\macd-auto\lib\site-packages\pkg_resources\__init__.py", line 1077, in obtain                                                                      
        return installer(requirement)                                                                                                                                                   
      File "d:\current_work\macd-auto\macd-auto\lib\site-packages\setuptools\dist.py", line 791, in fetch_build_egg                                                                     
        return cmd.easy_install(req)                                                                                                                                                    
      File "d:\current_work\macd-auto\macd-auto\lib\site-packages\setuptools\command\easy_install.py", line 673, in easy_install                                                        
        raise DistutilsError(msg)                                                                                                                                                       
    distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('incremental>=16.10.1')                                                                 
                                                                                                                                                                                        
    ----------------------------------------                                                                                                                                            
Command "python setup.py egg_info" failed with error code 1 in C:\Users\Shrey\AppData\Local\Temp\pip-install-xtwuif77\Twisted\                                                          
```

**Comment by bharatmicrosystems** [comment]
Do we have a plan to fix it yet?

**Comment by devrocksingh** [comment]
no updates on this issue? python 3.7 and above are increasingly becoming default

**Comment by vividvilla** [comment]
This issue was fixed on >=v3.9.0, upgrade and check.
